onnx_coreml/_operators.py

```diff
@@ -444,7 +445,7 @@ def get_coreml_axis(axes): # type: (List[int]) -> Text
     elif node.op_type == 'ReduceL2':
         mode = 'L2'
     elif node.op_type == 'ReduceLogSum':
-        mode = 'logsum'
+        return err.unsupported_op_configuration(builder, node, graph, "ReduceLogSum is not supported. Note: CoreML does support a logsum, but CoreML logsum computes sum(log(elements)), and ONNX defines logsum to be log(sum(elements))")
```

Comment:
that's weird but it's a good point

Reply:
Here, instead of erring out,  we should add two CoreML ops: reduce sum followed by unary log op.

Reply:
true, that would be a way to support it. Don't want to add this here because I want this PR to be the minimal effort to make things working again.

Reply:
Sure.